api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -33,11 +35,14 @@
 	private String name;
 	
 	private String description;
-	
+
+    @Deprecated
```

Comment:
The only compilation error you can get is if there is code directly referencing it which should be fixed